packages/fast-components-react-base/src/hypertext/hypertext.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, {HandledProps} from "../foundation";
+import {IHypertextHandledProps, IHypertextManagedClasses, IHypertextUnhandledProps} from "./hypertext.props";
+import {IHypertextClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+/* tslint:disable-next-line */
+class Hypertext extends Foundation<IHypertextHandledProps & IManagedClasses<IHypertextClassNameContract>,  React.AllHTMLAttributes<HTMLAnchorElement>, {}> {
+    protected handledProps: HandledProps<IHypertextHandledProps & IManagedClasses<IHypertextClassNameContract>> = {
+        managedClasses: void 0,
+    };
+
+    /**
+     * Generates class names
+     */
+    protected generateClassNames(): string {
+        return super.generateClassNames(this.props.managedClasses.hypertext);
+    }
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLAnchorElement> {
+        return (
+            <a
+                {...this.unhandledProps()}
+                className={this.generateClassNames()}
+                {...this.generateAttributes()}
+            >
+                {this.props.children}
+            </a>
+        );
+    }    
+
+    private generateAttributes(): {} {
+        const attributes = {};
+        const HREF_INDEX = "href";
+
+        if (this.props.href) {
+            attributes[HREF_INDEX] = this.props.href;
```

Comment:
does dot notation not work here? I know that you're likely getting an error, but if we can access via dot notation, let's do that instead.